commands/pull.js

```diff
@@ -30,6 +31,20 @@ function exec (cmd, opts = {}) {
   }
 }
 
+function spawn (cmd) {
+  const {spawn} = require('child_process')
+  return new Promise((resolve, reject) => {
+    let result = ''
+    let psql = spawn(cmd, [], {encoding: 'utf8', stdio: [ 'ignore', 'pipe', 'inherit' ], shell: true})
+    psql.stdout.on('data', function (data) {
+      result += data.toString()
+    })
+    psql.on('close', function (code) {
```

Comment:
@dickeyxxx I checked the old ruby code and they do not do anything with it so I kept the same behavior.  One thing I did notice was that the old one exits with `1` if the extensions do not match, which is something we do not do.

https://github.com/heroku/legacy-cli/blob/master/lib/heroku/helpers/pg_dump_restore.rb#L118
https://github.com/heroku/legacy-cli/blob/master/lib/heroku/helpers/pg_dump_restore.rb#L98